src/utils/getFloatValue.ts

```diff
@@ -1,4 +1,9 @@
 // https://gist.github.com/mold/42935cb1bdda7ae9b3ec72e9d0fa8666
```

Comment:
are there tests that can be written for these utilities?

Reply:
This is still very much in flux. @jasonvasquez and I have been discussing solutions for this over the last couple of days. Whatever the solution ends up, there will be tests for it. Should I remove this as the default in the meantime?